config/manager/manager.yaml

```
@@ -73,7 +73,8 @@ spec:
             memory: 500M
       - name: kube-rbac-proxy
         image: registry.redhat.io/openshift4/ose-kube-rbac-proxy:v4.13.0
-        securityContext:
+         imagePullPolicy: Always
+         securityContext:
```

the formatting looks weird, it looks like there is one space extra
```suggestion
        imagePullPolicy: Always
        securityContext:
```